### PR TITLE
reformat editlog.php

### PR DESCRIPTION
**Pull request by teiling88**

### CODE REVIEW

File: htdocs/editlog.php [hunk 1]
```diff
@@ -61,382 +57,470 @@
...
                         tpl_set_var('descMode', 2);
-                    else
-                    {
+                    } else {
                         // TinyMCE
                         $headers .= '<script language="javascript" type="text/javascript" src="resource2/tinymce/tiny_mce_gzip.js"></script>' . "\n";
-                        $headers .= '<script language="javascript" type="text/javascript" src="resource2/tinymce/config/log.js.php?logid=0&lang='.strtolower($locale).'"></script>' . "\n";
+                        $headers .= '<script language="javascript" type="text/javascript" src="resource2/tinymce/config/log.js.php?logid=0&lang=' . strtolower(
+                                $locale
+                            ) . '"></script>' . "\n";
```
Comment (following5) [inline htdocs/editlog.php]: Please remove an indent from the lines 461 and 462, as to PSR-2 rule: _Argument lists MAY be split across multiple lines, where each subsequent line is indented **once**._

File: htdocs/editlog.php [hunk 2]
```diff
@@ -61,382 +57,470 @@
...
+                    for ($i = 0; $i < count($smiley['show']); $i++) {
+                        if ($smiley['show'][$i] == '1') {
+                            $tmp_smiley = $smiley_link;
+                            $tmp_smiley = mb_ereg_replace('{smiley_image}', $smiley['image'][$i], $tmp_smiley);
+                            $tmp_smiley = mb_ereg_replace('{smiley_symbol}', $smiley['text'][$i], $tmp_smiley);
+                            $smilies = $smilies . '&nbsp;' . mb_ereg_replace(
+                                    '{smiley_file}',
+                                    $smiley['file'][$i],
+                                    $tmp_smiley
+                                ) . '&nbsp;';
```
Comment (following5) [inline htdocs/editlog.php]: Please remove an indent from the lines 488 to 491, as to PSR-2 rule: _Argument lists MAY be split across multiple lines, where each subsequent line is indented **once**._

**Comment by teiling88** [comment]
done

File: htdocs/editlog.php [hunk 3]
```diff
@@ -13,40 +13,36 @@
  *  Unicode Reminder メモ
  *****************************************************************************/
 
-require_once 'lib/consts.inc.php';
+require_once __DIR__ . 'lib/consts.inc.php';
```
Comment (following5) [inline htdocs/editlog.php]: There is a / missing between `__DIR__` and ' lib/...'.

But the `__DIR__` is not needed here, and there is no `__DIR__` in all the other toplevel scripts `require`s (except for mylists.php, see discussion there).

**Comment by following5** [comment]
don't like the inconsistent requires compared to all other scripts in the same dir, but that should not block the commit

**Comment by teiling88** [comment]
about this topic i created an internal thread to discuss that.

**Pull request merged**